INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
WARNING:googleapiclient.discovery_cache:file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
Traceback (most recent call last):
  File "c:\users\user\appdata\local\programs\python\python37\lib\site-packages\googleapiclient\discovery_cache\file_cache.py", line 33, in <module>
    from oauth2client.contrib.locked_file import LockedFile
ModuleNotFoundError: No module named 'oauth2client'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\user\appdata\local\programs\python\python37\lib\site-packages\googleapiclient\discovery_cache\file_cache.py", line 37, in <module>
    from oauth2client.locked_file import LockedFile
ModuleNotFoundError: No module named 'oauth2client'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\user\appdata\local\programs\python\python37\lib\site-packages\googleapiclient\discovery_cache\__init__.py", line 44, in autodetect
    from . import file_cache
  File "c:\users\user\appdata\local\programs\python\python37\lib\site-packages\googleapiclient\discovery_cache\file_cache.py", line 41, in <module>
    "file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth"
ImportError: file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
INFO:root:[{'kind': 'calendar#event', 'etag': '"3234220192614000"', 'id': '0gtl1ml1o44hh6ginck3rbisq1', 'status': 'confirmed', 'htmlLink': 'https://www.google.com/calendar/event?eid=MGd0bDFtbDFvNDRoaDZnaW5jazNyYmlzcTEgY182YzZ1ZTIwY3VnZmlzdjdyaDc5c2dlNzVnZ0Bn', 'created': '2021-03-30T12:57:28.000Z', 'updated': '2021-03-30T13:14:56.307Z', 'summary': 'Send 787445246', 'description': 'Hello Test', 'creator': {'email': 'randy@fightonmentality.com'}, 'organizer': {'email': 'c_6c6ue20cugfisv7rh79sge75gg@group.calendar.google.com', 'displayName': 'FOM - Paid Reminders', 'self': True}, 'start': {'dateTime': '2021-03-30T07:30:00-07:00'}, 'end': {'dateTime': '2021-03-30T08:30:00-07:00'}, 'iCalUID': '0gtl1ml1o44hh6ginck3rbisq1@google.com', 'sequence': 0, 'reminders': {'useDefault': True}, 'eventType': 'default'}]
INFO:werkzeug:127.0.0.1 - - [30/Mar/2021 18:59:54] "[37mGET /dashboard HTTP/1.1[0m" 200 -
INFO:werkzeug:127.0.0.1 - - [30/Mar/2021 18:59:55] "[37mGET /static/sms4_43518.ico HTTP/1.1[0m" 200 -
